bitcoin/src/psbt/mod.rs

```diff
@@ -1581,7 +1645,7 @@ mod tests {
             }],
         };
         let unknown: BTreeMap<raw::Key, Vec<u8>> =
-            vec![(raw::Key { type_value: 1, key_data: vec![0, 1] }, vec![3, 4, 5])]
+            vec![(raw::Key { type_value: 42, key_data: vec![0, 1] }, vec![3, 4, 5])]
```

Comment:
Deserialization will fail if `type_value = 1` because `Psbt::deserialize()` will attempt to deserialize this unknown value as a global xpub

Reply:
A separate patch for this too would make review easier. Us maintainers are a bit slack on the `psbt` module so stuff is not right there at the front of our memories. Please take pity on us.